docs/feature_backlight.md

```diff
@@ -119,7 +119,10 @@ The breathing effect is the same as in the hardware PWM implementation.
 |`backlight_step()`      |Cycle through backlight levels               |
 |`backlight_increase()`  |Increase the backlight level                 |
 |`backlight_decrease()`  |Decrease the backlight level                 |
-|`backlight_level(x)`    |Sets the backlight level to specified level  |
+|`backlight_level(x)`    |Sets the backlight level to specified 0-based|
```

Comment:
I think this is a little too verbose. Something like this reads better IMO:

```
Sets the backlight level, from 0 to `BACKLIGHT_LEVELS`
```

Reply:
I like it. Done.